File: domain/src/main/java/no/nav/melosys/domain/dokument/person/MidlertidigPostadresseNorge.java
package no.nav.melosys.domain.dokument.person;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlType;

@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "MidlertidigPostadresseNorge")
public class MidlertidigPostadresseNorge extends MidlertidigPostadresse {

    public String tilleggsadresse;

    public String tilleggsadresseType;

    public String poststed;

    public String bolignummer;

    public String kommunenummer;

    public Gateadresse gateadresse;
}

File: domain/src/main/kotlin/no/nav/melosys/domain/dokument/person/adresse/MidlertidigPostadresseNorge.kt
package no.nav.melosys.domain.dokument.person.adresse

import no.nav.melosys.domain.dokument.felles.Land
import no.nav.melosys.domain.dokument.felles.Periode
import java.time.LocalDateTime

class MidlertidigPostadresseNorge(
    var tilleggsadresse: String? = null,
    var tilleggsadresseType: String? = null,
    var poststed: String? = null,
    var bolignummer: String? = null,
    var kommunenummer: String? = null,
    var gateadresse: Gateadresse? = null,
    endringstidspunkt: LocalDateTime? = null,
    land: Land? = null,
    postleveringsPeriode: Periode? = null
) : MidlertidigPostadresse(endringstidspunkt, land, postleveringsPeriode)
